internal/converter/json/converter.go

```diff
@@ -166,47 +219,87 @@ func (f *FastJsonConverter) decodeArray(array []*fastjson.Value, field *ast.Json
 func (f *FastJsonConverter) decodeObject(obj *fastjson.Object, schema map[string]*ast.JsonStreamField) (map[string]interface{}, error) {
 	m := make(map[string]interface{})
 	for key, field := range schema {
+		if obj.Get(key) == nil {
```

Comment:
Generally, I prefer to do the type conversion/validation here together with decode just like protobuf decode. fastjson.Value has the type information and it is efficient to get the type. If we do it later, we will need reflection to get the real type, which is very slow.

Surely, it is OK to do this in the next PR